x-pack/test/apm_api_integration/tests/alerts/preview_chart_error_rate.spec.ts

```diff
@@ -12,22 +12,23 @@ import {
   TRANSACTION_TYPE,
 } from '@kbn/apm-plugin/common/es_fields/apm';
 import expect from '@kbn/expect';
-import archives from '../../common/fixtures/es_archiver/archives_metadata';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { generateData } from '../errors/generate_data';
```

Comment:
Please inline the data generation in the test, instead of importing. This way you can also adapt the data for this exact scenario.

Reply:
An example of inline data generation: https://github.com/elastic/kibana/blob/71f62fd85a503cfc9351d85b4681d33abf44918b/x-pack/test/apm_api_integration/tests/cold_start/cold_start.spec.ts#L116-L147

Reply:
updated